content/en/docs/components/central-dash/overview.md

```diff
@@ -16,19 +16,26 @@ following features:
 - A housing for the UIs of the components running in the cluster, including
   **Pipelines**, **Katib**, **Notebooks**, and more.
 - A [registration flow](/docs/components/central-dash/registration-flow/) that
-  prompts new users to set up their namespace if necessary. 
+  prompts new users to set up their namespace if necessary.
 
 ## Overview of Kubeflow UIs
 
 The Kubeflow UIs include the following:
 
-* **Home**, a central dashboard for navigation between the Kubeflow components.
-* **Pipelines** for a Kubeflow Pipelines dashboard.
-* **Notebook Servers** for Jupyter notebooks.
-* **Katib** for hyperparameter tuning.
-* **Artifact Store** for tracking of artifact metadata.
-* **Manage Contributors** for sharing user access across namespaces in the 
-  Kubeflow deployment.
+* **Home**:, Home, the central hub to access recent resources, active
+  experiments, and useful documentation.
+* **Notebook Servers**: To manage [Notebook servers](/docs/components/notebooks/).
+* **TensorBoards**: To manage TensorBoard servers.
+* **Models**: To manage deployed [KFServing models](/docs/components/kfserving/kfserving/).
+* **Volumes**: To manage the cluster's Volumes.
+* **Experiments (AutoML)**: To manage [Hyper Parameter tuning](/docs/components/katib/) experiments.
```

Comment:
```suggestion
* **Experiments (AutoML)**: To manage [Katib](/docs/components/katib/) experiments.
```

Reply:
+1

Reply:
@kimwnasptd it looks like we have some additional support for making this change. Would you mind pushing another commit?